controllers/article/ShareController.js

```diff
@@ -0,0 +1,70 @@
+import validator from 'validator';
+import eventBus from '../../helpers/eventBus';
+import db from '../../models';
+import HttpError from '../../helpers/exceptionHandler/httpError';
+
+const { Article, User } = db;
+/**
+ * Provides functionality for sharing articles via email
+ * @class ShareController
+ */
+class ShareController {
+  /**
+   * Validates the payload parameters passed to the mailto route
+   * @static
+   * @param {Object} req Request object
+   * @param {Object} res Response object
+   * @param {Function} next Next middleware
+   * @returns {null} Calls the next middleware
+   * @memberof ShareController
+   */
+  static async validateInputs(req, res, next) {
+    const { email: recipient } = req.body;
+    if (!recipient || !validator.isEmail(recipient)) {
+      return res.status(400).json({
```

Comment:
Nice job on your Implementation of this feature. I like the way you try to follow SRP. I would suggest you use the ```throwErrorIfNull()``` method and pass in the HTTP status code. What do you think?

Reply:
Thank you for the catch @blackshady . I will refactor the code to reuse the `throwErrorIfNull()` utility.